index.html

```diff
@@ -11,9 +11,18 @@
     <title>KTP Strikes Fall 2019</title>
   </head>
   <body>
-
-    <table id="table">
-      <thead>
+    <div id="title" class="py-3">
+      <h1 class="text-center">Kappa Theta Pi Strikes</h1>
+      <h5 class="text-center">A log of all strikes added and removed and their reasons why.</h5>  
+    </div>
+    <div id="loading-message" class="text-center mt-2">
+      <h5 class="py-2">Loading Table...</h5>
```

Comment:
Can you provide some context about these py-X classes? I'm not sure what they are, though I'm interested.

Reply:
Bootstrap has spacing classes for margin and padding. 'p' stands for padding, 'y' stands for top and bottom (y-axis), and the number is a multiplier to the $spacer bootstrap sass variable.
See: https://getbootstrap.com/docs/4.3/utilities/spacing/